src/generate-elements.ts

```diff
@@ -169,7 +180,8 @@ class ElementGatherer implements AnalysisVisitor {
       }
     } else {
       this._elementPaths.set(element, pathToElement);
-      const elem = serializeElementDescriptor(element, pathToElement);
+      const elem =
```

Comment:
having both 'element' and 'elem' identifiers in the same scope could be confusing? How are they different? Can that be in the name?